htdocs/admin/fckeditor.php

```diff
@@ -100,7 +100,7 @@
 		exit;
 	}
 	if ($action == 'disable_'.strtolower($const)) {
-		dolibarr_del_const($db, "FCKEDITOR_ENABLE_".$const, $conf->entity);
+		dolibarr_set_const($db, "FCKEDITOR_ENABLE_".$const, 0);
```

Comment:
You must keep the parameter $conf->entity into the dolibarr_set_const
If not, the constant will always be saved into the main company (this is used by the multicompany module to use 1 dolibarr to manage n companies).

Reply:
yes absolutely , i fix it